syntax = "proto2";
package dhc;


message rank_t {
    optional uint64 guid                = 1;
    repeated uint64 player_guid         = 2;
    repeated string player_name         = 4;
    repeated int32 player_icon          = 5;
    repeated int32 player_level         = 6;
    repeated int32 player_data          = 7;
    optional uint64 last_time           = 8;
}